### DISCUSSION TITLE
My input device defaults to background music, how can I use it only for output devices?

### DISCUSSION THREAD

**Discussion by Demianeen** [kyleneideck/BackgroundMusic#724] [Q&A]
I would like to use background music, but I found that each time I open a call. My input device defaults to background music which is super inconvenient. And it arises the question, why? As far as I understand it controls only output devices, isn't it?
![CleanShot 2024-03-09 at 15 52 55@2x](https://github.com/kyleneideck/BackgroundMusic/assets/51330172/b48f3c5c-4099-4a09-b158-36a1fe350138)

**Comment by gchilds** [comment]
The Background music device is a device with both input and output streams. Some [stupid] voip apps interpret you using an output device that also has input as meaning "use the input too". Ideally, I think, BGM would be split into two unidirectional devices.

**Reply by Demianeen** [reply]
As far as I see it is not only voip apps. In macOS settings, the default input device is set to background music (see the photo above)
